fe/fe-core/src/main/java/com/starrocks/catalog/StarOSAgent.java

```diff
@@ -148,6 +148,25 @@ public void addWorker(long backendId, String workerIpPort) {
         LOG.info("add worker {} success, backendId is {}", workerId, backendId);
     }
 
+    public void removeWorker(String workerIpPort) {
+        if (!workerToId.containsKey(workerIpPort)) {
```

Comment:
If FE has just started, maybe map is empty.
Should check if it exists in StarMgr, or directly drop from StarMgr.

Reply:
agree with @wyb.  I think drop directly is ok :)